misc/python/materialize/mzcompose/composition.py

```diff
@@ -788,7 +788,6 @@ def run(
             capture_and_print=capture_and_print,
             stdin=stdin,
             check=check,
-            environment=env_extra,
```

Comment:
another alternative would be something like:
```
environment={**os.environ, **env_extra},
```
this should ensure we retain whatever environment variables are already set

Reply:
Yes, that's pretty much what I did. It still works for me on macOS, so should be good